src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -256,4 +293,37 @@ protected virtual Task MapAsync(HttpContext httpContext, Route route, List<HttpC
         var aggregator = _factory.Get(route);
         return aggregator.Aggregate(route, httpContext, contexts);
     }
+
+    private static async Task CopyBufferToTargetRequestAsync(
+        DefaultHttpContext target,
+        Stream bufferedBody,
+        long? fromContentLength,
+        CancellationToken cancellationToken
+    )
+    {
+        if (fromContentLength is not null)
+        {
+            if (fromContentLength > int.MaxValue)
```

Comment:
This as no impact, as the request will be chunked if Buffer size > 655536 (https://github.com/ThreeMammals/Ocelot/blob/develop/src/Ocelot/Request/Mapper/StreamHttpContent.cs#L8)

Reply:
👍 
Would you rather me to just remove the check on int.MaxValue?